internal/azure/advisor/types.go

```diff
@@ -0,0 +1,88 @@
+package advisor
+
+import (
+	"time"
+)
+
+// AKSRecommendationSummary represents an Azure Advisor recommendation for AKS resources
+type AKSRecommendationSummary struct {
```

Comment:
for now, we don't need our own model for the output, just return what it got from cli is ok.